### PR TITLE
The shield generator's logging is now sorted by new to old order.

### PR DESCRIPTION
**Pull request by TorinoFermic**
## About The Pull Request

Quality of life code for Shield generator UI to prevent from losing track of this list as UI updates.

## Why It's Good For The Game

Helps to catch anyone who disabled the shield generator with a rogue computer or anything else like warnings.

## Changelog
:cl:
tweak: Shield Generator logging is sorted by new
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

Don't got a whole lot to say with this one. Just good QoL change.

Still needs code approval.

**Review by TheShown911 — APPROVED** [review]

**Comment by genessee596** [comment]
Delegate and maintainer approval. Merging now.

**Pull request merged**
